account_analytic_policy/README.rst

```diff
@@ -0,0 +1,10 @@
+Account Analytic Required
+=========================
+
+This module adds an option "analytic policy" on Accounts.
+
+You have the choice between 3 policies : always, never and optional. For
+example, if you want to have an analytic account on all your expenses, set
+the policy to "always" for the account that represent the "expense" ; then, if
+you try to save an account move line with an account of type "expense" without
+analytic account, you will get an proper error message.
```

Comment:
Please. Do not create a new file if there is a previous file existing.
Move the previous one to the new file name. `git move`

Preserve history while migrating modules, by doing this you are killing this module history track.

Reply:
I did a git mv, but as the chancge of the readme was in too many lines it is recognized as a new one. (it is a frequent situation).